langlib/lang.float/src/main/ballerina/float.bal

```diff
@@ -417,14 +607,19 @@ public isolated function toFixedString(float x, int? fractionDigits) returns str
 # But if `x` is NaN or infinite, the result will be the same as `value:toString`.
 # The digit before the decimal point will be zero only if all other digits
 # are zero.
-# This will panic if fractionDigits is less than 0.
+# This will panic if `fractionDigits` is less than 0.
```

Comment:
Did this change to make this consistent with spec.
Please refer this PR https://github.com/ballerina-platform/ballerina-spec/pull/1201